x-pack/legacy/plugins/reporting/export_types/csv/server/index.ts

```diff
@@ -1,22 +0,0 @@
-/*
```

Comment:
replaced all the `export_type/server/index.ts` files with `export_type/index.ts`